src-ts/proposalStage.ts

```diff
@@ -131,6 +133,124 @@ export enum ProposalStageStatus {
   TERMINATED = 4,
 }
 
+export class GovernorExecuteStage implements ProposalStage {
```

Comment:
Is this different from the governor queue stage? Cant we just use that?

Reply:
The nominee and member election have no timelock, so the execute method is different (and something needs to be done about status)

Reply:
thinking maybe having a single GovernorExecuteStage that determines the governor type (timelock, no timelock + vetting period, no timelock + no vetting period) and handles status/execute accordingly is cleanest?

Reply:
oh also getExecuteReceipt is different (looks for ProposalExecuted vs. CallScheduled).

Reply:
so idk